### DISCUSSION TITLE
Mealie on sidebar

### DISCUSSION THREAD

**Discussion by ilan212** [alexbelgium/hassio-addons#791] [General]
Hi,
i really love this thing! Thank you so much!
there are 2 things i wish it had.
1. Being able to add it to the home assistant sidebar.
2. Aligning text to the right for RTL languages but i guess that's a little outside your scope... Maybe someone else might know how to achieve that.

Been trying all sorts of things to add it to my dashboard or sidebar, at best i was able to use it from my computer never from the smartphone (Android - browser and companion app). Was wondering how do you guys use it.

**Comment by scooterama** [comment]
I just came here to plea for exactly this: mealie - add it to the home assistant sidebar

**Comment by alexbelgium** [comment]
Hi, a request is opened, but the app security makes it no compatible

**Reply by ilan212** [reply]
So it's something to do with the app itself and not anything with the way it's implemented... 
Wish i knew enough to even start to pretend to try and help. 

Think i was able to use it from a dashboard, but only locally though it has been a while since i used it and my memory isn't that good.

**Reply by alexbelgium** [reply]
Indeed it the past I had implemented that option, however changes in security that came later made it break the feature...
